website/client/components/ui/toggleSwitch.vue

```diff
@@ -1,6 +1,11 @@
 <template lang="pug">
 .clearfix.toggle-switch-container
   .float-left.toggle-switch-description {{ label }}
+  .float-left.svg-icon.inline.icon-16.info-icon(
+    v-if="infoId",
+    v-html="icons['info']",
+    :id="infoId",
```

Comment:
is an `:id` needed for the icon?

Reply:
That way, the pop up modal hovers on the icon